tests/test_dataset_generator.py

```diff
@@ -91,8 +95,256 @@ def test_generate_custom_dataset(self):
+                        "October 02 2006 22:34:32", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "August 19 2008 16:53:49",
+                    datetime.strptime(
+                        "August 19 2008 16:53:49", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 13 2010 17:18:44",
+                    datetime.strptime(
+                        "March 13 2010 17:18:44", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 11 2016 15:15:39",
+                    datetime.strptime(
+                        "March 11 2016 15:15:39", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "September 27 2018 18:24:03",
+                    datetime.strptime(
+                        "September 27 2018 18:24:03", self.date_format_list[0]
+                    ),
+                ],
+            ]
+        )
+
+        ordered_data = ordered_data[:, 0]
+        output_data = dg.get_ordered_column(data, "datetime", "ascending")
+
+        np.testing.assert_array_equal(output_data, ordered_data)
+
+    def test_get_ordered_column_datetime_descending(self):
+        data = dategen.random_datetimes(
+            rng=self.rng, start_date=self.start_date, end_date=self.end_date, num_rows=5
+        )
+
+        ordered_data = np.array(
+            [
+                [
+                    "September 27 2018 18:24:03",
+                    datetime.strptime(
+                        "September 27 2018 18:24:03", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 11 2016 15:15:39",
+                    datetime.strptime(
+                        "March 11 2016 15:15:39", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "March 13 2010 17:18:44",
+                    datetime.strptime(
+                        "March 13 2010 17:18:44", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "August 19 2008 16:53:49",
+                    datetime.strptime(
+                        "August 19 2008 16:53:49", self.date_format_list[0]
+                    ),
+                ],
+                [
+                    "October 02 2006 22:34:32",
+                    datetime.strptime(
+                        "October 02 2006 22:34:32", self.date_format_list[0]
+                    ),
+                ],
+            ]
+        )
+
+        ordered_data = ordered_data[:, 0]
+        output_data = dg.get_ordered_column(data, "datetime", "descending")
+
+        np.testing.assert_array_equal(output_data, ordered_data)
+
+    def test_get_ordered_column_custom_datetime_ascending(self):
+        custom_date_format = ["%Y %m %d"]
```

Comment:
we should probably have multiple values here ... to validate it processes correctly if it is more than one value in the list